gui/wxpython/vdigit/toolbars.py

```diff
@@ -882,6 +946,43 @@ def StartEditing(self, mapLayer):
             else:
                 return
 
+        # check and clean zero length lines, boundaries
+        zero_length = self._topoZeroLengthCheck(
+            map_name=mapLayer.GetName(),
+        )
+
+        if zero_length is None:
+            return
```

Comment:
The one return statement above returns None, but the three down below return boolean. Pylint should give a warning about that.